pkg/corerp/api/v20231001preview/environment_conversion_test.go

```diff
@@ -269,6 +269,88 @@ func TestConvertVersionedToDataModel(t *testing.T) {
 			},
 			err: nil,
 		},
+		{
+			filename: "environmentresource-terraformrecipe-http.json",
+			expected: &datamodel.Environment{
+				BaseResource: v1.BaseResource{
+					TrackedResource: v1.TrackedResource{
+						ID:   "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/radius-test-rg/providers/Applications.Core/environments/env0",
+						Name: "env0",
+						Type: "Applications.Core/environments",
+						Tags: map[string]string{},
+					},
+					InternalMetadata: v1.InternalMetadata{
+						CreatedAPIVersion:      "2023-10-01-preview",
+						UpdatedAPIVersion:      "2023-10-01-preview",
+						AsyncProvisioningState: v1.ProvisioningStateAccepted,
+					},
+				},
+				Properties: datamodel.EnvironmentProperties{
+					Compute: rpv1.EnvironmentCompute{
+						Kind: "kubernetes",
+						KubernetesCompute: rpv1.KubernetesComputeProperties{
+							ResourceID: "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/testGroup/providers/Microsoft.ContainerService/managedClusters/radiusTestCluster",
+							Namespace:  "default",
+						},
+					},
+					Providers: datamodel.Providers{
+						Azure: datamodel.ProvidersAzure{
+							Scope: "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/testGroup",
+						},
+					},
+					Recipes: map[string]map[string]datamodel.EnvironmentRecipeProperties{
+						ds_ctrl.MongoDatabasesResourceType: {
+							"cosmos-recipe": datamodel.EnvironmentRecipeProperties{
+								TemplateKind: recipes.TemplateKindTerraform,
+								TemplatePath: "https://example.com/vpc-module.zip",
+							},
+						},
+					},
+				},
+			},
+			err: nil,
+		},
+		{
+			filename: "environmentresource-terraformrecipe-tfregistry.json",
```

Comment:
Same here, should this already be covered by this existing scenario? https://github.com/radius-project/radius/blob/9b47d5ea6407e451381fcf8ce971de51f63bae90/pkg/corerp/api/v20231001preview/environment_conversion_test.go#L131